src/ScheduleFunctions.cpp

```diff
@@ -1112,7 +1112,8 @@ Stmt schedule_functions(const vector<Function> &outputs,
             debug(1) << "Injecting realization of " << order[i-1] << '\n';
             InjectRealization injector(f, is_output, target);
             s = injector.mutate(s);
-            internal_assert(injector.found_store_level && injector.found_compute_level);
+            internal_assert(injector.found_store_level && injector.found_compute_level)
```

Comment:
I think this error should have been caught during schedule validation with a better error message. This internal assertion is just a consistency check, and there are many types of bug that could trigger it, not just the condition described.